src/client/datascience/notebook/notebookControllerManager.ts

```diff
@@ -112,9 +112,7 @@ export class NotebookControllerManager implements INotebookControllerManager, IE
         // Sign up for document either opening or closing
         this.notebook.onDidOpenNotebookDocument(this.onDidOpenNotebookDocument, this, this.disposables);
         // If the extension activates after installing Jupyter extension, then ensure we load controllers right now.
-        if (this.isLocalLaunch) {
-            this.notebook.notebookDocuments.forEach((notebook) => this.onDidOpenNotebookDocument(notebook).catch(noop));
-        }
+        this.notebook.notebookDocuments.forEach((notebook) => this.onDidOpenNotebookDocument(notebook).catch(noop));
```

Comment:
Change makes sense, do we know why it was gated on isLocalLaunch in the first place? That doesn't seem relevant to me. Just making sure we're not regressing on some reason this was only local before.

Reply:
Back then I wasn't sure how remote worked, and didn't want to break remote.
I think I fixed this at the last minute before a release, hence didn't want to make unnecessary changes